.ci/StarterRuleSet-AllRulesByCategory.groovy.txt

```diff
@@ -333,7 +333,10 @@ ruleset {
     PackageNameMatchesFilePath
     ParameterName
     PropertyName
-    VariableName
+    VariableName {
+        finalRegex = null
+        ignoreVariableNames = 'SEPARATOR'
+    }
```

Comment:
CodeNarc was giving a violation-
`
Violation: Rule=VariableName P=2 Line=8 Msg=[Variable named SEPARATOR in class None does not match the pattern [a-z][a-zA-Z0-9]*] Src=[@Field final static String SEPARATOR = System.getProperty("file.separator")]
`

Flagging fields using `@Field` annotation is a bug, it was corrected in the later version of CodeNarc, see https://github.com/CodeNarc/CodeNarc/blob/master/CHANGELOG.md#version-210----apr-2021, https://github.com/CodeNarc/CodeNarc/issues/561